### PR TITLE
Add on GE21-M-L1 and GE21-P-L2 plots

### PR DESCRIPTION
**Pull request by Dongwoon77**
**PR description:**

- replace GE21-M-L2 with GE21-M-L1 
- Lumi-based chamber status, OptoHybrid status and VFAT status (chamber vs. VFAT)
- Add on GE21-M-L1 recohits occupancy and average cluster plots
- Add on GE21-M AMC status plot. 

This PR is related to PRs on cms-sw; see  
https://github.com/cms-sw/cmssw/pull/41915 (to master)  
https://github.com/cms-sw/cmssw/pull/42483 (backport)

### CODE REVIEW

**Comment by arooshap** [comment]
Hi @Dongwoon77,

your changes have been incorporated in HG2402b release and deployed in the testbed VMs. Kindly validate, thanks!

**Comment by Dongwoon77** [comment]
Dear @arooshap.

Sure, I'll test it with #1304 PR.

File: dqmgui/layouts/gem-layouts.py [hunk 1]
```diff
@@ -30,53 +30,51 @@ def GEMLayout(i, p, *rows): i["GEM/Layouts/" + p] = DQMItem(layout=rows)
...
+GEMLayout(dqmitems, 'Common/05 GE21-M-L1-M1 Lumi-based chamber status', 
+    [{'path': 'GEM/EventInfo/chamberStatus_inLumi_GE21-M-L1-M1', 'description': 'Lumi-based chamber status in GE21-M-L1-M1'}], 
     )
 
 
 
-GEMLayout(dqmitems, 'Common/06 GE21-P-L2-M2 Lumi-based chamber status', 
-    [{'path': 'GEM/EventInfo/chamberStatus_inLumi_GE21-P-L2-M2', 'description': 'Lumi-based chamber status in GE21-P-L2-M2'}], 
+GEMLayout(dqmitems, 'Common/06 GE21-M-L1-M2 Lumi-based chamber status', 
+    [{'path': 'GEM/EventInfo/chamberStatus_inLumi_GE21-M-L1-M2', 'description': 'Lumi-based chamber status in GE21-M-L1-M2'}],
```
Comment (nothingface0) [inline dqmgui/layouts/gem-layouts.py:40]: @Dongwoon77 Is this correct? When we check inside the ROOT files created by the DQM GEM client we still see `chamberStatus_inLumi_GE21-P-L2-M2`: 

image

Comment (arooshap) [reply dqmgui/layouts/gem-layouts.py:40]: @Dongwoon77 please confirm this so that we can proceed with the deployment.

**Comment by Dongwoon77** [comment]
Hi @arooshap @nothingface0.

This link is some of the lines matching to GE21-P-L2 module [1]

I didn't edit that, but this lines were pushed back at code so It looks like difference at git-diff.

and no plots for GE21-M-L1 are not implemented in the central P5 DQM currently, but it will be added in the coming runs.

[1] https://github.com/Dongwoon77/deployment/blob/GEMOnlineDQMForFullGE21/dqmgui/layouts/gem-layouts.py#L56-L75

Thank you!

**Comment by nothingface0** [comment]
> Hi @arooshap @nothingface0.
> 
> This link is some of the lines matching to GE21-P-L2 module [1]
> 
> I didn't edit that, but this lines were pushed back at code so It looks like difference at git-diff.
> 
> and no plots for GE21-M-L1 are not implemented in the central P5 DQM currently, but it will be added in the coming runs.
> 
> [1] [Dongwoon77/deployment@`GEMOnlineDQMForFullGE21`/dqmgui/layouts/gem-layouts.py#L56-L75](https://github.com/Dongwoon77/deployment/blob/GEMOnlineDQMForFullGE21/dqmgui/layouts/gem-layouts.py?rgh-link-date=2024-03-14T03%3A46%3A45Z#L56-L75)
> 
> Thank you!
@Dongwoon77 Just to make sure, this means that this PR is fine as it is for now?

**Comment by Dongwoon77** [comment]
@nothingface0 

Yep, we have checked that this PR looks fine.

Thank you!

**Pull request merged**